crates/uv-resolver/src/pubgrub/priority.rs

```diff
@@ -178,9 +178,7 @@ impl PubGrubPriorities {
                 // The ConflictEarly` match avoids infinite loops.
                 if matches!(
                     entry.get(),
-                    PubGrubPriority::ConflictLate(_)
-                        | PubGrubPriority::ConflictEarly(_)
-                        | PubGrubPriority::Singleton(_)
```

Comment:
@konstin -- Heads up, I removed this... Otherwise, the `wrong_backtracking_basic` test failed. I think it succeeded by accident before? Since all the conflicts are in singletons.